cypress/e2e/awx/views/schedules.cy.ts

```diff
@@ -78,7 +78,7 @@ describe('Schedules - Create and Delete', () => {
       cy.navigateTo('awx', 'schedules');
       cy.verifyPageTitle('Schedules');
       cy.getByDataCy('create-schedule').click();
-      cy.verifyPageTitle('Create Schedule');
+      cy.verifyPageTitle('Create schedule');
```

Comment:
this was already addressed in [here](https://github.com/ansible/ansible-ui/pull/2754/files#diff-6710eea15d62a82a05d60f79fceb32cf813b84e6929970b2dc9b90243d0aad6cR81)